resources/styles/global/forms.less

```diff
@@ -33,6 +38,27 @@ textarea {
   }
 }
 
+.invalid-form {
```

Comment:
Hmm, what about `is-` prefixes for classes conveying boolean states? (like `is-valid`, `is-required`, and `is-empty`?)